cmd/terramate/cli/telemetry/endpoint.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 Terramate GmbH
+// SPDX-License-Identifier: MPL-2.0
+
+//go:build !localhostEndpoints
+
+package telemetry
+
+import (
+	"net/url"
+)
+
+func Endpoint() url.URL {
+	var u url.URL
+	u.Scheme = "https"
+	u.Host = "analytics.terrmate.io"
```

Comment:
```suggestion
	u.Host = "analytics.terramate.io"
```

Reply:
Thanks for spotting. Fixed in [720d370](https://github.com/terramate-io/terramate/pull/1981/commits/720d37021530b4e72694414563606c30a6f54df8)